ColumnReinforcement/SingleTieMultipleRebars.py

```diff
@@ -604,8 +666,24 @@ def editSingleTieMultipleRebars(
     """
     if len(rebar_group.RebarGroups) == 0:
         return rebar_group
-    if hasattr(rebar_group.RebarGroups[0], "Ties"):
-        Tie = rebar_group.RebarGroups[0].Ties[0]
+    for i, tmp_rebar_group in enumerate(rebar_group.RebarGroups):
+        if hasattr(tmp_rebar_group, "Ties"):
+            if len(tmp_rebar_group.Ties) > 0:
+                Tie = tmp_rebar_group.Ties[0]
+                break
+            else:
+                print(
```

Comment:
why you used print here? Print statement not print in FreeCAD console. Use https://github.com/amrit3701/FreeCAD-Reinforcement/blob/master/Rebarfunc.py#L284 for warning.

Reply:
Done as suggested above:
commit: https://github.com/SurajDadral/FreeCAD-Reinforcement/commit/bbdc797d90cf41f556d18241d35f9d9bbcfdf2ed